lib/bracket-matcher-view.coffee

```diff
@@ -13,6 +13,14 @@ endPairMatches =
   ']': '['
   '}': '{'
 
+pairClasses =
+  '(': 'bracket-matcher-parentessis-open'
```

Comment:
Generally the language packages call `(` `brace.round`, so `round-brace` might be a better name here